dev/ci/ci-pidetop.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+# $0 is not the safest way, but...
+ci_dir="$(dirname "$0")"
+. "${ci_dir}/ci-common.sh"
+
+pidetop_CI_DIR="${CI_BUILD_DIR}/pidetop"
+
+git_checkout "${pidetop_CI_BRANCH}" "${pidetop_CI_GITURL}" "${pidetop_CI_DIR}"
+
+( cd "${pidetop_CI_DIR}" && coq_makefile -f Make -o Makefile.top && make -f Makefile.top "-j${NJOBS}" && make install-toploop -f Makefile.top )
+
+# Test the toploop can be loaded by Coq (grep init detects if it was loaded,
+# the || true is there because 'xxx' is not a valid message and we run with -o
+# pipefail)
+echo xxx | (coqtop -main-channel stdfds -toploop pidetop || true) | grep init
```

Comment:
I am not sure if I like this test, WDYT @SkySkimmer ?

Reply:
It may help to know what is wrong with it, in your opinion.

Reply:
Is there some reason we shouldn't use a valid message?

Reply:
I can try to see if there is an equivalent of `Quit.` (now that I think at it, there should be a `Terminate` one).